src/compiler/crystal/command.cr

```diff
@@ -585,4 +590,8 @@ class Crystal::Command
     @color = false if ARGV.includes?("--no-color") || ENV["TERM"]? == "dumb"
     Crystal.error msg, @color, exit_code: exit_code
   end
+
+  private def use_crystal_opts
+    @options = ENV.fetch("CRYSTAL_OPTS", "").split.concat(options)
```

Comment:
Is this a declaration that nobody will ever need to use spaces within options to Crystal compiler?

Would be nice to support shell quoting.

Reply:
Good catch. Definitely an improvement.

Reply:
Is there a safe already way to split that respecting shell quoting?

Reply:
No. We'd need to port this from Ruby: https://ruby-doc.org/stdlib-2.5.1/libdoc/shellwords/rdoc/Shellwords.html (or do something similar)